Big Sur pyenv build issue

I wonder if the update from Dec. 14th caused this.

```
❯ make setup-pyenv
python-build: use openssl@1.1 from homebrew
python-build: use readline from homebrew
Downloading Python-3.6.10.tar.xz...
-> https://www.python.org/ftp/python/3.6.10/Python-3.6.10.tar.xz
Installing Python-3.6.10...
python-build: use readline from homebrew
python-build: use zlib from xcode sdk

BUILD FAILED (OS X 11.1 using python-build 20180424)

Inspect or clean up the working tree at /var/folders/zr/vhvlh_t57vz0_qkfwh9fmlm00000gn/T/python-build.20201216114654.83571
Results logged to /var/folders/zr/vhvlh_t57vz0_qkfwh9fmlm00000gn/T/python-build.20201216114654.83571.log

Last 10 log lines:
        ret = sendfile(in, out, offset, &sbytes, &sf, flags);
              ^
./Modules/posixmodule.c:10432:5: warning: code will never be executed [-Wunreachable-code]
    Py_FatalError("abort() called from Python code didn't abort!");
    ^~~~~~~~~~~~~
1 warning generated.
clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -I/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include   -I/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include   -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-missing-field-initializers -Wstrict-prototypes   -I. -I./Include -I/usr/local/opt/readline/include -I/usr/local/opt/readline/include -I/usr/local/opt/openssl@1.1/include -I/Users/armenzg/.pyenv/versions/3.6.10/include  -I/usr/local/opt/readline/include -I/usr/local/opt/readline/include -I/usr/local/opt/openssl@1.1/include -I/Users/armenzg/.pyenv/versions/3.6.10/include   -DPy_BUILD_CORE  -c ./Modules/pwdmodule.c -o Modules/pwdmodule.o
1 warning and 1 error generated.
make[1]: *** [Modules/posixmodule.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [setup-pyenv] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@ WEBPACK := yarn build-acceptance
 # Currently, this is only required to install black via pre-commit.
 REQUIRED_PY3_VERSION := $(shell grep "3.6" .python-version)
 
-BIG_SUR := $(shell sw_vers -productVersion | egrep "11\.0\.")
+BIG_SUR := $(shell sw_vers -productVersion | egrep "11\.")
 
 bootstrap: develop init-config run-dependent-services create-db apply-migrations build-platform-assets
 

